gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -142,15 +144,14 @@ def remove_chrome_from_icon_grids():
         log.info("Checking %s", path)
         try:
             with open(path, 'r') as f:
-                grid = f.read()
+                grid = json.load(fp=f)
 
-            if GOOGLE_CHROME in grid:
+            if replace_chrome_with_chromium(grid[DESKTOP_GRID_ID]):
                 log.info("Updating %s", path)
-                grid = grid.replace(dq(GOOGLE_CHROME), dq(CHROMIUM_BROWSER))
 
                 with open(path, 'w') as f:
```

Comment:
It isn't a part of this commit, but it'd be nice to avoid overloading `f` like this

Reply:
I don't really agree: this is outside the scope of the first `with` block.

Reply:
The second `with` block on line 151 is nested within the first on line 144.

...Unless my eyes are playing tricks on me?

Reply:
They are! The body of the first `with` is a single line. Then a new `if` block appears.

```python
with open(path, 'r') as f:
    grid = json.load(fp=f)

if replace_chrome_with_chromium(grid[DESKTOP_GRID_ID]):
    log.info("Updating %s", path)

    with open(path, 'w') as f:
        json.dump(grid, fp=f)
```

Reply:
I see - GitHub must have conveniently left out that `if` statement by suppressing context. Consider my comment irrelevant then :-)